analysis/report_templates/default.html

```diff
@@ -153,14 +153,56 @@ <h5 class="center-align">By avg. rank</h5>
             <ul class="collapsible">
                 <li>
                     <div class="collapsible-header">
-                        Median {{ experiment.type }}-coverages on each benchmark
+                        Median relative code-coverages on each benchmark
                     </div>
                     <div class="collapsible-body">
-                        {{ experiment.summary_table.to_html() }}
+                        {{ experiment.relative_code_summary_table.render() }}
+                      <ul>
+                          <li> Fuzzers are sorted by "FuzzerMean" (average median relative coverage), highest on the left.</li>
+                          <li> Green background = highest relative median coverage.</li>
+                          <li> Blue gradient background = greater than 95% relative median coverage.</li>
+                          <li> *note: <code>relative_coverage  = trial_coverage / experiment_max_coverage</code></li>
```

Comment:
If you keep the current semantics of the table, this is the most important i think, so consider starting the description with this instead of making it a note in the end.

Reply:
Putting the description at the top without extra spacing creates something like this.
![image](https://user-images.githubusercontent.com/558033/112041096-5ff26b80-8b1c-11eb-8351-78504cb52af3.png)

After adding some spacing, it should look like this:
![image](https://user-images.githubusercontent.com/558033/112051683-a1891380-8b28-11eb-85a1-95f926a0a959.png)